linux/Dockerfile

```
@@ -197,6 +200,14 @@ RUN curl -LO https://releases.hashicorp.com/vault/${VAULT_VERSION}/${VAULT_FILEN
     ./aws/install && \
     rm -rf aws awscliv2.zip
 
+# Install Pulumi
+RUN curl --retry 10 -fsSL https://get.pulumi.com/ | bash -s -- --version $PULUMI_VERSION && \
+    mv ~/.pulumi/bin/* /usr/bin
```

Installing files out of `apt` control should now happen in essential system directories. I think the following would be less puzzling:
```suggestion
    mv ~/.pulumi/bin/* /usr/local/bin
```